sw/device/silicon_creator/manuf/lib/provisioning_functest.c

```diff
@@ -23,11 +23,6 @@
 
 OTTF_DEFINE_TEST_CONFIG(.enable_uart_flow_control = true);
 
-static_assert(
-    (sizeof(wrapped_rma_unlock_token_t) % sizeof(uint32_t)) == 0,
-    "Size of the wrapped_rma_unlock_token_t struct must the an even multiple "
-    "of 32-bit words since it is written to the non-volatile flash region.");
-
```

Comment:
Should this have been changed to `sizeof(uint64_t)` instead of removed as per https://github.com/lowRISC/opentitan/pull/18404#discussion_r1188692669?

Reply:
No, it should be removed, because this data is no longer written to flash before a reset, it is written to the retention SRAM, where these constraints no longer apply. This change was made after that comment, so the comment is outdated.

Reply:
thanks!